webpack.config.node.js

```diff
@@ -101,13 +105,7 @@ const webpackConfig = {
 			{
 				test: /\.jsx?$/,
 				exclude: /(node_modules|devdocs[\/\\]search-index)/,
-				use: [
-					{
-						loader: 'thread-loader',
-						options: { workers: 3 },
```

Comment:
Have you verified this is actually faster? In HappyPack (what we had before `thread-loader`), 3 threads was the sweet spot. More than that was barely faster and even slower, because of the interprocess communication overhead.

Reply:
I only tested it locally (4 core / 8 threads / 2.9 GHz Intel Core i7) and it appeared to be slightly faster with 8 threads than 3, but there's also a fair bit of noise. 

The real hope here is to make it more adaptive, not just faster. For instance, the build machines have 16 cores, not the 2, 4 or 8 that we typically have on dev laptops, though it's unclear how many are being allocated to each docker instance.  On dev machines that only have 2 cores, forcing 3 doesn't really make sense. I'm wary of making a static core count decision.